recipes/emsdk/all/conanfile.py

```diff
@@ -0,0 +1,139 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import json
+
+required_conan_version = ">=1.33.0"
+
+
+class EmSDKConan(ConanFile):
+    name = "emsdk"
+    description = "Emscripten SDK. Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("emsdk", "emscripten", "sdk")
+    license = "MIT"
+    settings = "os", "arch"
+
+    short_paths = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("nodejs/16.3.0")
+        # self.requires("python")  # FIXME: Not available as Conan package
+        # self.requires("wasm")  # FIXME: Not available as Conan package
+
+    def validate(self):
+        if hasattr(self, "settings_target") and self.settings_target:
+            if self.settings_target.os != "Emscripten":
+                raise ConanInvalidConfiguration("When using {}, target os must be Emscripten".format(self.name))
+
+    def package_id(self):
+        del self.info.settings.os
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    @staticmethod
+    def _chmod_plus_x(filename):
+        if os.name == "posix":
+            os.chmod(filename, os.stat(filename).st_mode | 0o111)
+
+    def _tools_for_version(self):
+        ret = {}
+        # Select release-upstream from version (wasm-binaries)
+        with open(os.path.join(self.build_folder, self._source_subfolder, "emscripten-releases-tags.json"), 'r') as f:
+            data = json.load(f)
+            ret['wasm'] = "releases-upstream-{}-64bit".format(data["releases"][self.version])
+        # Select python and node versions
+        with open(os.path.join(self.build_folder, self._source_subfolder, "emsdk_manifest.json"), 'r') as f:
+            data = json.load(f)
+            tools = data["tools"]
+            python = next((it for it in tools if (it["id"] == "python" and not it.get("is_old", False))), None)
+            ret["python"] = "python-{}-64bit".format(python["version"])
```

Comment:
```suggestion
            if self.settings.os == "Linux":
                python = next((it for it in tools if (it["id"] == "python" and not it.get("is_old", False))), None)
                ret["python"] = "python-{}-64bit".format(python["version"])
```

Looks like Python is not installed by `emsdk` in Linux: https://emscripten.org/docs/getting_started/downloads.html#linux